pkg/npm/api/tsconfig.json

```diff
@@ -5,7 +5,7 @@
 	  "outDir": "./dist",
     "module": "ES2020",
     "noImplicitAny": true,
-    "target": "ES2020",
+    "target": "ES2017",
```

Comment:
Any particular reason for this? Not complaining just curious

Reply:
I was getting incompatibility with the create-urbit-app defaults. Maybe it's my setup that's the problem, but all of this (incl the pull here: https://github.com/tylershuster/cra-template-urbit/pull/1) was just what it took to get the fresh CRA-urbit app running for me.